src/auth/auth.controller.ts

```diff
@@ -9,12 +9,13 @@ import {
 } from '@nestjs/swagger';
 import { AccessToken } from '../common/interfaces/access-token.interface';
 import { User } from '@prisma/client';
+import { IsUserVerifiedGuard } from '../user/guards/is-user-verified.guard';
 
 @Controller('/auth')
 export class AuthController {
   constructor(private authService: AuthService) {}
 
-  @UseGuards(AuthGuard('local'))
+  @UseGuards(AuthGuard('local'), IsUserVerifiedGuard)
```

Comment:
Since all endpoints should be blocked for unverified users by default, this could be done in your local strategy implementation.